### DISCUSSION TITLE
230724 FE 회의를 위한 회의(19:35~20:35)

### DISCUSSION THREAD

**Discussion by ukkodeveloper** [woowacourse-teams/2023-shook#107] [회의]
### 안건
현재 FE 회의에서 서로가 많이 지치는 모습을 보였습니다. 도밥의 용기로 이 같은 내용이 수면 위로 올라왔고 앞으로 남은 기간 동안 원활한 회의를 위해 "회의를 위한 회의"를 하려고 합니다.

월요일 12시에 회의 진행을 원하시면 이모지를 남겨주세요. 급하게 discussion 올린 거라 12시가 힘들면 월요일 오후나 화요일에라도 꼭 회의를 진행했으면 좋겠습니다.

### 목적
원활하고 효율적인 회의 진행을 이어갈 수 있도록 FE 그라운드 룰을 정한다.

### 준비 사항
1. 지금껏 회의에서 본인을 힘들었던 이유를 준비해주세요. (지금까지 코드 컨벤션 회의, 개발환경 회의, 피그마 시안 회의, 글로벌 스타일 회의를 떠올리면서 준비하시면 될 것 같아요.)
2. 해당 원인에 대한 해결책으로는 어떤 것들이 있을 지 준비해주세요.

#### 우코의 예시
- 문제1: 글로벌 스타일을 정하는 회의 때는 빠르게 의사결정을 했으면 했는데 하나씩 학습하고 있었고 그 때문에 시간 지연되어서 답답했어요.
- 개선1: 앞으로는 회의의 목적을 명확히 하는 것이 좋겠어요! 회의 시작 전에 5분동안 회의 목적에 대해서 이야기하면서 목적 달성을 위해 회의가 어떻게 진행되어야 하는지 이야기 나눴으면 좋겠어요.
- 문제2: 컨벤션을 정하는 회의에서 특정 코드의 옳고 그름을 판단하고 있는 제 자신을 목격했어요. 그리고 이전 코드와 리뷰를 찾아보는 제 행위가 회의를 지연시키는 원인이었습니다.
- 개선2: 앞으로는 회의의 목적이 `학습`인지 `의사결정`인지 명확하게 유형을 밝히는 것이 좋을 것 같아요. 그리고 회의에 학습적인 내용(백그라운드)가 포함되어 있다면, `사전에 학습해 올 내용`을 명시했으면 좋겠어요. (우리가 회의 도중에 자료를 찾는 시간이 너무 긴 것 같더라구요.)

### 진행 사항
1. 회의 시작 5분동안 회의 목표에 대해서만 이야기 합니다. [해당 글의 1번 참고](https://blog.jandi.com/ko/2016/11/18/time-management-tips-meeting/)
3. 회의에서 아쉬웠던 점을 이야기 합니다.
4. 아쉬웠던 점을 토대로 어떻게 원활한 회의를 할 수 있을 지 논의합니다.

### ⌛ 예상 소요 시간
1시간 (1시간을 넘을 경우 회의를 바로 종료할 예정입니다. 1시간안에 무조건 회의 결과물을 내어야 한다는 마음으로 임했으면 합니다.)

**Comment by jariita** [comment]
✅

**Comment by ukkodeveloper** [comment]
# 회의 목표
- 목적: 각자 아쉬웠던 점을 통해 이후 회의 개선. 룰 정하기
## 회의
# 아쉬운 점
## 우코
- 문제1: 글로벌 스타일을 정하는 회의 때는 빠르게 의사결정을 했으면 했는데 하나씩 학습하고 있었고 그 때문에 시간 지연되어서 답답했어요.
- 개선1: 앞으로는 회의의 목적을 명확히 하는 것이 좋겠어요! 회의 시작 전에 5분동안 회의 목적에 대해서 이야기하면서 목적 달성을 위해 회의가 어떻게 진행되어야 하는지 이야기 나눴으면 좋겠어요.
- 문제2: 컨벤션을 정하는 회의에서 특정 코드의 옳고 그름을 판단하고 있는 제 자신을 목격했어요. 그리고 이전 코드와 리뷰를 찾아보는 제 행위가 회의를 지연시키는 원인이었습니다.
- 개선2: 앞으로는 회의의 목적이 학습인지 의사결정인지 명확하게 유형을 밝히는 것이 좋을 것 같아요. 그리고 회의에 학습적인 내용(백그라운드)가 포함되어 있다면, 사전에 학습해 올 내용을 명시했으면 좋겠어요. (우리가 회의 도중에 자료를 찾는 시간이 너무 긴 것 같더라구요.)

## 코난
- 상황 1. 프론트엔드 환경설정할 때 이미 한번 논의 되었고, 구체적인 것도 각자 찾아보고 서로 물어봤었음. 그리고 위키까지 정리가 되어 있었다. 그래서 트러블 슈팅만 예상하고 있었는데, 회의에서 갑자기 디깅하기 시작했다. 그래서 시간이 오래 걸렸다. 2시간 예상했는데 6시간 걸렸다.
- 해결책: (룰) 회의에서 디깅하지 않는다.
- 해결책: 트러블 슈팅 면에서 회의가 길어진다면,  별도의 회의로 분리할 필요가 있다.

- 상황 2. 장단점을 이야기하는 과정있었다. 반박에 대한 반박이 이어지는 경우가 많았다.
- 해결책: 어차피 하나의 장점에는 단점이 있다. 반박에 대한 반박을 하기보다는 특정 논의에 대해 "장단점"을 정리하는 식으로 하자. (어떤 사항에는 장단점이 있기 마련!)
- 해결책: 상대 의견에 대한 질문을 3~5분을 넘기지 않는다. 그 이상의 의문점이나 반박점은 개인적으로 이야기 한다. (회의에서 하지 않고 개인적으로 풀 수 있다. 그러다 팀적으로 중요한 사항이 논의되고 있으면 해당 사항은 팀 안건으로 올린다.)

## 도밥
- 문제: 글로벌 스타일 정하는 회의에서, 팀원들이 지친 것이 보였다. 회의에서 조금 더 팀원들끼리 논의하고 글로벌 스타일을 정하고 싶었다. **다들 영혼이 없어 보였다**. 특히 theme으로 설정한 것이 자동완성 안되어서 타입을 지정할 필요가 있었다. 근데 그때 너무 팀원들이 굳어 있었다. 팀원들이 이게 중요치 않은 건지, 무엇인지 헷갈렸다.
- 해결책: (회의뿐만 아니라 상황 상 사람들이 힘들어 하는 상황일 수 있음.) 체크인 시에 서로의 집중력 정도를 공유하면 좋을 것 같다. 이를 반영하여 유동적으로 회의 시간을 조절할 수도 있을 것 같다.
- 함께 진행할 사항인지, 개인이 진행할 사항인지 의사결정을 할 필요가 있다. 

# 구체적인 그라운드 룰
1. 회의 진행 전 5분 동안 회의 목적, 진행, 어떤 결과가 있을 지, 예상 종료 시간을 상기하고 회의를 시작한다.
2. 회의에서 디깅하지 않는다. (스터디를 통해 디깅하도록 한다.)
4. 특정 논의가 길어지면 별도의 회의로 분리한다. (누구나, 언제든 회의 도중에 발언해서!)
5. 상대 의견에 대한 질문이 3분이 넘어가면, 장단점만 정리하고 다음 논의로 넘어간다.
6. 회의하고 싶은 내용이 생기면, 즉석에서 10분정도 함께 논의 후 함께 진행할 필요가 있는지 정한다.
7. discussion에 회의를 올릴 때 회의 목적, 진행, 어떤 결과가 있을 지, 예상 종료 시간, 사전에 학습/준비해 올 내용을 명시한다. (참고 자료 등)

**Reply by Cyma-s** [reply]
우리 프론트 짱이다~ 👯
